src/write_gcm.c

```diff
@@ -2416,10 +2416,14 @@ static char *wg_get_from_metadata_server(const char *base, const char *resource,
   char buffer[2048];
   if (wg_curl_get_or_post(buffer, sizeof(buffer), url, NULL, headers,
       num_headers) != 0) {
-    INFO("write_gcm: wg_get_from_metadata_server failed fetching %s", url);
+    INFO("write_gcm: wg_get_from_metadata_server failed to fetch %s metadata "
+         "from %s", provider, url);
     return NULL;
+  } else {
+    INFO("write_gcm: wg_get_from_metadata_server received response from %s "
```

Comment:
Being that you're breaking out of the if statement above with the return, is there any reason to not keep it as it was before, outside of the condition entirely? Is it clearer to have the `} else {`?

Reply:
N/A after refactoring. PTAL.